### PR TITLE
update joinus.PNG and the readme docs

### PR DESCRIPTION
**Pull request by D-DanielYang**
update joinus.PNG and the readme docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,49 +79,54 @@ English | [简体中文](README_ch.md)
 
 ### Object Detection
 - Pedestrian detection, vehicle detection, and more industrial-grade ultra-large-scale pretrained models are provided.
+- Many thanks to CopyRight@[PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection) for the pre-trained models, you can try to train your models with PadddleDetection.
 <div align="center">
 <img src="./docs/imgs/Readme_Related/Image_ObjectDetection_Pedestrian_Vehicle.gif"  width = "642" height = "400" />
 </div>
 
 ### Key Point Detection
 - Supports body, face and hands key point detection for single or multiple person.
```
Comment (haoyuying) [inline README.md]: 改成Sopport吧

File: README.md [hunk 2]
```diff
@@ -170,6 +178,7 @@ English | [简体中文](README_ch.md)
 
 ### Video Classification
 - Short video classification trained via large-scale video dataset, supports 3000+ tag types prediction.
```
Comment (haoyuying) [inline README.md]: Short form video 怎么样

File: README.md (hunk 1 above)
Comment (D-DanielYang) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (D-DanielYang) [reply README.md]: done

**Pull request merged**